examples/horizontal_convection.jl

```diff
@@ -0,0 +1,384 @@
+# # Horizontal convection example
+#
+# In "horizontal convection", a non-uniform buoyancy is imposed on top of an initially resting fluid.
+#
+# This example demonstrates:
+#
+#   * How to use `ComputedField`s for output
+#   * How to post-process saved output
+#
+# ## Install dependencies
+#
+# First let's make sure we have all required packages installed.
+
+# ```julia
+# using Pkg
+# pkg"add Oceananigans, JLD2, Plots"
+# ```
+
+# ## Horizontal convection
+#
+# We'll consider here the two-dimensional verion of horizontal convection in which a flow
```

Comment:
version?

Reply:
version of what? Oceananigans? Any version… :) The example will be continuously run and build with the docs so if you can see it in the docs then you can run it with that version. If there is a problem the doc builds fails and we don’t merge or publish a new tagged version anyway.

Reply:
Ok I see. I had misunderstood completely. Indeed “version “ can be dropped!

Reply:
```suggestion
# We'll consider here the two-dimensional version of horizontal convection in which a flow
```

Reply:
I meant there's a typo in "verion"!

I couldn't figure out how to suggest a commit to fix that so I just typed one word and was completely unhelpful haha sorry!

Reply:
Haha

Reply:
To suggest a change you click on the comment text box, which brings up a prompt:

![image](https://user-images.githubusercontent.com/15271942/126024432-0de660b9-2448-4aa6-90d6-17e5b8edf565.png)

I've circled the button for suggesting a change. You can also highlight several lines and suggest changes to more than one line at once.